packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -474,16 +483,21 @@ export const usePermissions = createPermissionHook<
 
 // @needsAudit
 /**
- * __Available only on iOS >= 14.__ Allows the user to update the assets that your app has access to.
+ * __Available only on iOS >= 14 and Android >= 14.__ Allows the user to update the assets that your app has access to.
  * The system modal is only displayed if the user originally allowed only `limited` access to their
  * media library, otherwise this method is a no-op.
- * @return A promise that either rejects if the method is unavailable (meaning the device is not
- * running iOS >= 14), or resolves to `void`.
+ * @return A promise that either rejects if the method is unavailable, or resolves to `void`.
```

Comment:
```suggestion
 * @param mediaTypes [FILL_ME_OUT]
 * @return A promise that either rejects if the method is unavailable, or resolves to `void`.
```

Let's add new param description.